src/settings.py

```diff
@@ -47,7 +47,7 @@ class GlobalSettings:
     cluttered_table_place_goal_conditioned_sampling = True
 
     # repeated nextto env parameters
-    repeated_nextto_num_dots = 25
+    repeated_nextto_num_dots = 15
```

Comment:
@tomsilver made this change for speed, everything is way too slow without it (even prediction error)

Reply:
because we're trying to ground large operators in the beginning?

Reply:
yup